text/label/subword_nmt/apply.py

```diff
@@ -106,39 +108,68 @@ class ApplyBPEToTextJob(Job):
     Apply BPE codes on a text file
     """
 
-    def __init__(self, text_file, bpe_codes, bpe_vocab=None, subword_nmt_repo=None):
+    __sis_hash_exclude__ = {"gzip_output": False}
+
+    def __init__(
+        self,
+        text_file: tk.Path,
+        bpe_codes: tk.Path,
+        bpe_vocab=None,
+        subword_nmt_repo=None,
+        gzip_output=False,
+    ):
         """
         :param Path text_file: words text file to convert to bpe
-        :param Path bpe_codes: bpe codes file
-        :param Path|None bpe_vocab: if provided, then merge operations that produce OOV are reverted
-        :param Path|str|None subword_nmt_repo: subword nmt repository path. see also `CloneGitRepositoryJob`
+        :param Path bpe_codes: bpe codes file, e.g. ReturnnTrainBpeJob.out_bpe_codes
+        :param Path|None bpe_vocab: if provided, then merge operations that produce OOV are reverted,
+            use e.g. ReturnnTrainBpeJob.out_bpe_apply_vocab
+        :param Path|None subword_nmt_repo: subword nmt repository path. see also `CloneGitRepositoryJob`
         """
```

Comment:
even though its trivial, comment for `gzip_output` please.

Reply:
Done